dune/grid/cpgrid/OrientedEntityTable.hpp

```diff
@@ -162,6 +164,36 @@ namespace Dune
 		return super_t::operator==(other);
 	    }
 
+            /** @brief Prints the relation matrix corresponding to the table, sparse format.
+
+             Let the entities of codimensions f and t be given by
+             the sets \f$E^f = { e^f_i } \f$ and \f$E^t = { e^t_j }\f$.
+             A relation matrix R is defined by
+	     \f{eqnarray*}{
```

Comment:
> Probably, but I do not know for sure

I don't know what the "probably" refers to, so I can't reply intelligently.  However, our [`cmake/Templates/Doxyfile`](https://github.com/OPM/dune-cornerpoint/blob/master/cmake/Templates/Doxyfile#L1147) does include the request
```
# The EXTRA_PACKAGES tag can be to specify one or more names of LaTeX
# packages that should be included in the LaTeX output.

EXTRA_PACKAGES         = amsmath
```
so using the previously suggested `cases` construction should be possible.  Would you mind trying that?

In general `eqnarray` should not be used at all in any documents, but it seems impossible to eliminate due to old habits.